src/01-Static-HTML/02-Static-HTML-unscaffolded.purs

```diff
@@ -0,0 +1,62 @@
+module StaticHTML.StaticHTML where
+
+module Main where
+
+import Prelude 
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Data.Const (Const)
+
+import Halogen as H
+import Halogen.Aff as HA
+import Halogen.VDom.Driver (runUI)
+import Halogen.HTML as HH
+import Halogen.HTML (HTML)
+
+type Query = Const Unit
+type Input = Unit
+type State = Unit
+type Output = Void
+data Action = Unit
+
+-- | Shows how to use Halogen VDOM DSL to render HTML without properties or CSS
+staticHtml :: StaticHTML
+staticHtml =
+  HH.div_
+    -- The 'div' tag takes an Array of children
+    [ HH.div_
+      [ HH.span_
+        -- as does the `span` tag
+        [ HH.text "This is text in a span!" ]
+      ]
+    , HH.button_
+      [ HH.text "You can click me, but I don't do anything." ]
+    ]
+
+render :: State -> HTML (H.ComponentSlot HTML () Aff Action) Action
+render _ =
+  staticHtml
+
+handleAction :: Action -> H.HalogenM State Action () Void Aff Unit
+handleAction _ = pure unit
```

Comment:
maybe this should say `handleAction _ = pure initialState` because we're not using state at all in this example.

Reply:
Yeah, I realised that afterward :)

This process (of making a PR / giving feedback) is pretty interesting because I'm not fully clear on what I'm trying to get at yet, so I apologise for you having to come along on this journey with me while I work out what the issues are, but hopefully it's beneficial to us both, and others.

I suppose the core issue I have is that the docs and examples in the docs don't clearly explain what all the pieces are in one place, so your document has this decided advantage of being able to isolate complexity... yet it seems the material has a few gaps, as you point out in #42 — once we've gone through the simple examples, maybe we need a point where they can decide to either continue with more examples and slowly introduce the more tricky/real-world types (for example, adding state, then adding passed-in state (ie `input`), then when actions come into the picture) then parents, then children, then passing state between them, then querying, then parents & children components, slowly lifting the training wheels off as we go?